resources/dicts/events/lifegen_events/murder.json

```diff
@@ -418,7 +428,7 @@
     ],
     "murder elder medicinecatapprentice": [
         "If you were to be honest, you much preferred the eldest medicine cat's care. However, they're out of the den today. Instead, you have the inexperienced v_c to help your care. Old age had left you with a few health problems, like bellyaches, but herbs would put these symptoms at bay. As you bite into your herbs today, something tastes unusually foul. You spit them out quickly, but v_c calls you out for being dramatic. They even eat the herb bundle themselves to prove it to you! Turns out, v_c mistook water hemlock for parsley, and you watch with little pity as their body stills. That could have been you.",
-        "As v_c pads up to you, asking for you to accompany them to gather herbs, you press down a scowl. Did they only ask you since you were an elder, and thus your time was less important than other cats? Nevertheless, you accompany the young apprentice, and out of the corner of your eye, you spot a bush of bright, red berries. You innocently ask v_c what berries these are, and as the inexperienced medicine cat sniffs the plants curiously, you declare that you're pretty sure these are raspberries, and ask the apprentice to taste to make sure. v_c takes one, and before long, you are carrying v_c home on your back, deathberry juice dripping from their mouth."
+        "As v_c pads up to you, asking for you to accompany them to gather herbs, you press down a scowl. Did they only ask you since you were an elder, and thus your time was less important than other cats? Nevertheless, you accompany the young apprentice, and out of the corner of your eye, you spot a bush of bright, red berries. You innocently ask v_c what berries these are, and as the inexperienced medicine cat sniffs the plants curiously, you declare that you're pretty sure these are raspberries, and ask the apprentice to taste to make sure. v_c takes one, and before long, you are carrying the dead body of v_c back to camp."
```

Comment:
"Did they only ask you since you were an elder, and thus your time was less important than other cats?"
to
"Did they only ask you since you're an elder, your time being less important than the others..?"

"You innocently ask v_c what berries these are, and as the inexperienced medicine cat sniffs the plants curiously, you declare that you're pretty sure these are raspberries, and ask the apprentice to taste to make sure. v_c takes one, and before long, you are carrying the dead body of v_c back to camp."
to
"You innocently ask v_c what type of berries these are. As the inexperienced apprentice sniffs the plant curiously, you hum and haw, before declaring you're pretty sure these are raspberries. v_c crunches a berry between their teeth to make sure, and before long, you're carrying their body back to camp."